src/main/java/org/jabref/gui/libraryproperties/general/GeneralPropertiesViewModel.java

```diff
@@ -61,6 +66,16 @@ public void setValues() {
         selectedDatabaseModeProperty.setValue(metaData.getMode().orElse(BibDatabaseMode.BIBLATEX));
         generalFileDirectoryProperty.setValue(metaData.getDefaultFileDirectory().orElse("").trim());
         userSpecificFileDirectoryProperty.setValue(metaData.getUserFileDirectory(preferencesService.getFilePreferences().getUserAndHost()).orElse("").trim());
+        userSpecificFileDirectoryProperty.setValue(metaData.getUserFileDirectory(preferencesService.getFilePreferences().getUserAndHost()).map(path -> usernameProperty.getValue() + ": " + path).orElse("").trim());
+        String username = preferencesService.getFilePreferences().getUserAndHost();   // get the username
+        usernameProperty.setValue(username);
+        try {
+            InetAddress localHost = InetAddress.getLocalHost();
+            hostProperty.set(localHost.getHostName()); // get the host Name
```

Comment:
Remove this code - JabRef already knows the host name. It is returned in `getUserAndHost()`.